qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -176,30 +172,25 @@ def _check_duration(duration: Optional[float] = None) -> float:
     return duration
 
 
-def new_ramsey_sequence(duration=None, pre_post_rotation=False, **kwargs):
+def new_ramsey_sequence(duration=1, pre_post_rotation=False, name=None):
```

Comment:
Shall we just set name=`ramsey` at the end while creating the `DynamicDecouplingSequence` object, and remove this input?

Reply:
Let's keep it, that way behaviour stays the same